user.js

```diff
@@ -400,12 +400,12 @@ user_pref("loop.logDomains",					false);
 //user_pref("app.update.auto",		true);
 
 // PREF: Enable blocking reported web forgeries
-// http://kb.mozillazine.org/Browser.safebrowsing.enabled
+// https://wiki.mozilla.org/Security/Safe_Browsing
 // http://kb.mozillazine.org/Safe_browsing
 // https://support.mozilla.org/en-US/kb/how-does-phishing-and-malware-protection-work
 // http://forums.mozillazine.org/viewtopic.php?f=39&t=2711237&p=12896849#p12896849
 // CIS 2.3.4
-user_pref("browser.safebrowsing.enabled",			true);
+user_pref("browser.safebrowsing.phishing.enabled",		true);
```

Comment:
Perhaps the 2 prefs should be kept with a comment noting which FF version is affected? (At least while we still supprt FF versions >= 45, see https://github.com/pyllyukko/user.js/issues/86#issuecomment-287873061):

```
user_pref("browser.safebrowsing.enabled",			true);  // (<50)
user_pref("browser.safebrowsing.phishing.enabled",		true); // (>=50)
```